### DISCUSSION TITLE
Import Entries as Drafts

### DISCUSSION THREAD

**Discussion by Kashkin** [engram-design/FeedMe#1094] [Ideas]
At present is possible to import content that is Enabled, Disabled, Expired, or Pending.

There is no ability to import Entries as Drafts.

We have a number of projects where FeedMe imports are treated as the seed for content, and must be edited or reviewed before being published. At present, we import them as disabled, but this doesn't have the same advantages of importing drafts:

- **Clearer states**
Is something disabled because it's freshly imported or because it's been published and then disabled?
- **Soft validation**
Invalid states could be imported and then rectified by an author.
- **Placeholder slug**
An author can easily change the title and the slug before publication.

There might be others, but that's what I've got so far.

How feasible is this feature, post Craft 4?

**Comment by samlising** [comment]
I also support this request. Additionally to be able to import content that updates as drafts would also be very useful. 

The use case would be if content is already live, but when importing updated content the only options are to update and keep live without the chance to review or update and disable where the content is not available anymore. Having the ability to update as a draft would allow content to stay live and have a review process before publishing a draft.

I hope this feature can be added.
